api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -2672,7 +2672,7 @@
         <field>
           <name>property</name>
           <version>4.0.0+</version>
-          <description>Specifies that this profile will be activated when this system property is
+          <description>Specifies that this profile will be activated when this property is
             specified.</description>
```

Comment:
We can't use project property for profile activation - right? 
It can be user or system property.
Maybe it should be clear here?

Reply:
This needs to be checked...

Reply:
I checked `PropertyProfileActivator`. It DOES use user and then system properties. No project/model properties, *but* the passed `ProfileActivationContext` contains a `getProjectProperties()` method so we could make this happen.